.travis.yml

```diff
@@ -17,5 +17,5 @@ after_success:
 
 env:
 - ESVER=-
-- ESVER=0.90.11
 - ESVER=1.4.2
```

Comment:
@clintonb: Is there a reason why we didn't drop 1.4.2 as well if moving to 1.5.2?